### PR TITLE
Stylistic Changes to Password Strength Meter

### PR DESCRIPTION
**Pull request by teh48th**
This is a stylistic change to make the Strength Meter to be more presentable. This is not tied to any open issues.

**Old**

password_strength_old

**New**

password_strength_new

### CODE REVIEW

**Comment by gtiwari333** [comment]
@teh48th The Strength Meter in my app was showing like the one you mentioned as "**Old**" . It turns out somehow main.css was missing from my Index.html page.

**Comment by deepu105** [comment]
This needs to be checked, it was already working like what you show as new.  Ill check next week

File: generators/client/templates/src/main/webapp/app/account/password/_password-strength-bar.directive.js [hunk 1]
```diff
@@ -88,6 +88,10 @@
                         .css({ 'background-color': '#DDD' })
                         .slice(0, c.idx)
                         .css({ 'background-color': c.col });
+                }else {
```
Comment (jdubois) [inline generators/client/templates/src/main/webapp/app/account/password/_password-strength-bar.directive.js]: formatting is not good, a space misses

**Comment by jdubois** [comment]
Might depend on your browser??
I'm sorry but it already works as expected for me, so I'm closing it.

**Pull request closed without merging**